djredcap/management/subcommands/inspect.py

```diff
@@ -60,7 +60,23 @@ def handle(self, fileName=None, *args, **options):
         fin.seek(0)
         reader = csv.DictReader(fin, fieldnames=header_keys, dialect=dialect)
 
-        reader.next()
+        
+        #
+        # Open the file in a normal reader to see if the file is a REDCap data dictionary
+        # A REDCap data Dictionary begins with the four necessary columns:
+        # Variable/Field Name | Form Name | Field Type | Field Label in any order
+        #
+        header = reader.next()
+
+        if header['field_name'] != 'Variable / Field Name' and
+            header['form_name'] != 'Form Name' and
+            header['field_name'] != 'Field Name' and
+            header['field_label'] != 'Field Label':
+
+            print('Invalid input. file must be a REDCap data dictionary, NOT the data itself')

```

Comment:
It is more appropriate to write this to `stderr`. Also the message makes an assumption that *it is* a data file, but we only validated that *it is not* a data dictionary. I recommend using this message:

```python
sys.stderr.write('Invalid header. File must be a valid REDCap data dictionary.\n')
```